uint8 axis
---
uint32 encoder_error